lib/zendesk_apps_tools/command.rb

```diff
@@ -28,9 +28,17 @@ class Command < Thor
     desc 'new', 'Generate a new app'
     method_option :'iframe-only', type: :boolean,
                                   default: false,
-                                  desc: 'Create an iFrame Only app template',
-                                  aliases: ['-i', '--v2']
+                                  hide: true,
+                                  aliases: ['--v2']
+    method_option :v1, type: :boolean, default: false, desc: 'Create a version 1 app template'
     def new
+      if options[:'iframe-only']
+        if options[:v1]
+          say_error_and_exit "Apps can't have both v1 and v2 set."
+        else
+          say_status 'warning', 'v2 is the default for new apps, the --v2 option has no effect.', :yellow
```

Comment:
This warning will display if `--iframe-only` is explicitly used? Maybe put:

> v2 is the default for new apps, the --v2 and --iframe-only options have no effect.

?